### ISSUE TITLE
Latest stable `ring` (0.12.1) does not compile with nightly rust

### ISSUE THREAD

**Issue by stepancheg** [briansmith/ring#607]
```
$ git co 0.12.1
$ rm -rf target Cargo.lock
$ cargo +nightly build
    Updating registry `https://github.com/rust-lang/crates.io-index`
   Compiling either v1.4.0
   Compiling untrusted v0.5.1
   Compiling scopeguard v0.3.3
   Compiling rayon-core v1.3.0
   Compiling lazy_static v0.2.11
   Compiling libc v0.2.34
   Compiling gcc v0.3.54
   Compiling coco v0.1.1
   Compiling rand v0.3.18
   Compiling num_cpus v1.7.0
   Compiling rayon v0.8.2
   Compiling ring v0.12.1 (file:///Users/nga/devel/left/ring)
warning: lint fat_ptr_transmutes has been removed: was accidentally removed back in 2014
  --> build.rs:23:5
   |
23 |     fat_ptr_transmutes,
   |     ^^^^^^^^^^^^^^^^^^
   |
   = note: #[warn(renamed_and_removed_lints)] on by default

error[E0583]: file not found for module `montgomery`
  --> src/arithmetic/arithmetic.rs:15:9
   |
15 | pub mod montgomery;
   |         ^^^^^^^^^^
   |
   = help: name the file either arithmetic/montgomery.rs or arithmetic/montgomery/mod.rs inside the directory "src/arithmetic"

error: aborting due to previous error

error: Could not compile `ring`.

To learn more, run the command again with --verbose.

% rustc +nightly --version
rustc 1.24.0-nightly (4a7c072fa 2017-12-25)
```

**Comment by stepancheg** [comment]
I see it was fixed in #598.

Could you please release `0.12.2` update with the fix?

**Comment by briansmith** [comment]
> Could you please release 0.12.2 update with the fix?

The way to get the fix is to upgrade to 0.13. Rustls will probably release a new major version release soon, and that release can use the 0.13 series.

Cheers,
Brian
